docs/en/operations/settings/settings.md

```diff
@@ -404,6 +404,38 @@ Possible values:
 
 Default value: 0.
 
+## partial_merge_join_optimizations {#partial_merge_join_optimizations}
+
+Enables of disables optimizations in partial merge join algorithm for [JOIN](../../sql-reference/statements/select/join.md) queries.
```

Comment:
We should describe that this setting is temporal. It enables some improvements that could possibly lead to wrong results. So it's a way to disable them. Even more it could ontain different set of optimisations in different versions.